Description: Don't install the binary file that we've stripped from the tarball.
 The binary is removed due to lack of compliance with the dfsg
Author: Steve Conklin <steve@conklinhouse.com>
Last-Update: 2011-02-18

--- a/setup.py
+++ b/setup.py
@@ -93,7 +93,6 @@
                     ('/usr/share/d-rats/forms', form_files),
                     ('/usr/share/d-rats/images', image_files),
                     ('/usr/share/d-rats/ui', ui_files),
-                    ('/usr/share/d-rats/libexec', ["libexec/lzhuf"]),
                     ('/usr/share/man/man1', man_files),
                     ('/usr/share/doc/d-rats', ['COPYING']),
                     ] + locale_files)
